paper/paper.md

```diff
@@ -38,12 +38,12 @@ bibliography: paper.bib
 
 # Summary
 PyMilo is an open-source Python package that addresses the limitations of existing machine learning (ML) model storage formats by providing a transparent, reliable, end-to-end, and safe method for exporting and deploying trained models. 
-Current tools rely on black-box or executable formats that obscure internal model structures, making them difficult to audit, verify, or safely share. 
+Current tools rely on black-box or executable formats that obscure internal model structures, making them difficult to audit, verify, or safely share. Meanwhile, tensor-centric formats such as SafeTensors [@huggingface_safetensors_2022] securely store and transfer numerical tensors but do not capture the symbolic or structural composition of classical machine-learning models (e.g., scikit-learn pipelines), which remain PyMilo’s primary focus. 
 Others apply structural transformations during export that may degrade predictive performance and reduce the model to a limited inference-only interface. 
-In contrast, PyMilo serializes models in a transparent human-readable format that preserves end-to-end model fidelity and enables reliable, safe, and interpretable exchange. 
+In contrast, PyMilo serializes models in a transparent human-readable format that preserves end-to-end model fidelity and enables reliable, safe, and interpretable exchange. Here, transparent refers to the ability to inspect model internals through a human-readable structure without execution, and end-to-end fidelity denotes that a model exported and re-imported with PyMilo retains the exact same signature, functionality, parameters, and internal structure as the original, ensuring complete behavioral and structural equivalence.
 This package is designed to make the preservation and reuse of trained ML models safer, more interpretable, and easier to manage across different stages of the ML workflow (\autoref{fig:overall}).
 
-![PyMilo is an end-to-end, transparent, and safe solution for transporting models from machine learning frameworks to the target devices. PyMilo preserves the original model's structure while transferring, allowing it to be imported back as the exact same object in its native framework.\label{fig:overall}](pymilo_outlook.png)
+![PyMilo is an end-to-end, transparent, and safe solution for transporting models from machine learning frameworks to the target devices. PyMilo preserves the original model's structure while transferring, allowing it to be imported back as the exact same object in its native framework. Currently (v1.3), PyMilo supports models built with scikit-learn. Support for PyTorch and TensorFlow is planned in upcoming releases.\label{fig:overall}](pymilo_outlook.png)
```

Comment:
I think we should update the figure as well.

Reply:
Currently, I believe it is not necessary to update this figure, as we have outlined our plan to support them directly.